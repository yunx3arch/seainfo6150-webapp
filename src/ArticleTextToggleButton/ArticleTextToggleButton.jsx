import React from "react";
import style from "./ArticleTextToggleButton.module.css";


const ArticleTextToggleButton=(props)=>{
    return (
    <button className={style.button}
    onClick={props.clicked}>
        {props.buttonText}
        </button>
    );
};
export  default ArticleTextToggleButton;