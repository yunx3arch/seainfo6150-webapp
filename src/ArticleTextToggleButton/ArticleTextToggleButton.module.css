.button{
    color:antiquewhite;
    background-color: rgb(250, 87, 87);
    padding: 10px;
    border-radius: 20%;
}