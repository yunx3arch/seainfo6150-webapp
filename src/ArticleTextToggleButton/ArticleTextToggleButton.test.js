import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";
import { fireEvent } from "@testing-library/react";

describe("ArticleTextToggleButton tests", () => {
  it("renders correctly", () => {
    const onClick = jest.fn();
    const { container } = render(<ArticleTextToggleButton buttonText={"Show more"} onClick={onClick}/>);
    expect(container).toMatchSnapshot();
  });
});