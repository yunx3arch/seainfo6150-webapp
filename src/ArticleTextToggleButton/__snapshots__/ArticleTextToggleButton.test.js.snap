// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTextToggleButton tests renders correctly 1`] = `
<div>
  <button
    class="button"
  >
    Show more
  </button>
</div>
`;
